runtime/bootstrap.go

```diff
@@ -43,6 +43,7 @@ type Bootstrap struct {
 	ToWeaveletFile *os.File // Pipe to send to weavelet (weavertest only).
 	ToEnvelopeFile *os.File // Pipe to send to envelope (weavertest only).
 	TestConfig     string   // Config file contents (weavertest only).
+	Quiet          bool     // Don't log or print anything.
```

Comment:
Will this be something we want to provide to non-test weaver runs as well?

Also, I wonder if the default should be quiet with extra logging only if the user asks for it. A downside is that we might not print out useful URLs? But perhaps weaver status commands fix that problem?

Reply:
I think it's only useful for single process weavertests. Right now, non-test weaver runs only show messages explicitly logged by the user. System logs aren't shown by default. They have to use `weaver logs --system` to see those. 

The one exception is the little rolodex that gets printed right when the program starts. 

```
╭───────────────────────────────────────────────────╮
│ app        : collatz                              │
│ deployment : 9d029072-ef9e-4c3d-b03f-e6e92afda1ad │
╰───────────────────────────────────────────────────╯
```

I added this because the user needs some way to learn the deployment id. Otherwise when they run things like `weaver status`, they aren't sure which deployment is which. We could provide the deployment id via some API and not print it out though.

Reply:
Okay. Let's leave as is for now.